Bearded.Utilities/SpaceTime/2d/Acceleration2.cs

```diff
@@ -131,8 +131,13 @@ public Acceleration2(Acceleration x, Acceleration y)
 
         public override string ToString() => ToString(null, CultureInfo.CurrentCulture);
 
-        public string ToString(string? format, IFormatProvider? formatProvider)
-            => $"({value.X.ToString(format, formatProvider)}, {value.Y.ToString(format, formatProvider)}) u/t²";
+        public string ToString(string? format, IFormatProvider? formatProvider) => "(" +
+            $"{value.X.ToString(format, formatProvider)}, " +
+            $"{value.Y.ToString(format, formatProvider)}) u/t²";
```

Comment:
Here and some others below: I would prefer if we didn't change the existing implementations unless we actually intend their behaviour to change (or if there is a really good reason - is there?), so that the change set is easier to understand.